frontend/store/surplus.ts

```diff
@@ -0,0 +1,35 @@
+import Vue from 'vue';
+import type { SurplusAuctionFullData } from 'auctions-core/src/types';
+import { ActionContext } from 'vuex';
+import { fetchActiveSurplusAuctions } from 'auctions-core/src/surplus';
+
+interface State {
+    auctionStorage: Record<string, SurplusAuctionFullData>;
+}
+
+const getInitialState = (): State => ({
+    auctionStorage: {},
+});
+
+export const state = (): State => getInitialState();
+
+export const getters = {
+    auctionStorage(state: State) {
+        return state.auctionStorage;
+    },
+};
+
+export const mutations = {
+    addAuctionToStorage(state: State, auction: SurplusAuctionFullData) {
```

Comment:
Wouldn't auction added to the storage contain `state`? Then, the type should be `SurplusAuction`